bears/vcs/git/GitCommitBear.py

```diff
@@ -269,7 +336,9 @@ def check_issue_reference(self, body,
                               body_close_issue: bool=False,
                               body_close_issue_full_url: bool=False,
                               body_close_issue_on_last_line: bool=False,
-                              body_enforce_issue_reference: bool=False):
+                              body_enforce_issue_reference: bool=False,
+                              body_enforce_issue_open: bool=False,
```

Comment:
I would personally like a setting which only enforces this for the last line close/fix reference, since it is not uncommon to include references such as "Related to `old issue`" in the body.